File: 3046-split-the-array/3046-split-the-array.cpp
class Solution {
public:
    bool isPossibleToSplit(vector<int>& nums) {
        unordered_map<int,int> mp;
        for(int i=0;i<nums.size();i++) {
            mp[nums[i]]++;
        }
        for(auto it=mp.begin();it!=mp.end();it++) {
        if(it->second>2) {
            return false;
        }
        }
        return true;
        
    }
};

File: 3046-split-the-array/3046-split-the-array.java
class Solution {
    public boolean isPossibleToSplit(int[] nums) {
        int arr[] = new int[101];
         for(int i=0;i<nums.length;i++){
             arr[nums[i]]++;
         }
        for(int i : arr){
            if(i>2){
                return false;
            }
        }
        return true;
    }
}